src/main/scala/common/Tile.scala

```diff
@@ -370,13 +370,57 @@ class ShuttleTileModuleImp(outer: ShuttleTile) extends BaseTileModuleImp(outer)
           ptwPortId += 1
         }
         rocc.module.io.cmd <> cmdRouter.io.out(i)
-        rocc.module.io.mem := DontCare
-        rocc.module.io.mem.req.ready := false.B
-        assert(!rocc.module.io.mem.req.valid)
         respArb.io.in(i) <> Queue(rocc.module.io.resp)
         rocc.module.io.fpu_req.ready := false.B
         rocc.module.io.fpu_resp.valid := false.B
         rocc.module.io.fpu_resp.bits := DontCare
+
+        dcachePorts(i + 2).req.valid := rocc.module.io.mem.req.valid
+        dcachePorts(i + 2).req.bits.addr := rocc.module.io.mem.req.bits.addr
+        dcachePorts(i + 2).req.bits.tag := rocc.module.io.mem.req.bits.tag
+        dcachePorts(i + 2).req.bits.cmd := rocc.module.io.mem.req.bits.cmd
+        dcachePorts(i + 2).req.bits.size := rocc.module.io.mem.req.bits.size
+        dcachePorts(i + 2).req.bits.signed := rocc.module.io.mem.req.bits.signed
+        dcachePorts(i + 2).req.bits.data := rocc.module.io.mem.req.bits.data
+        dcachePorts(i + 2).req.bits.mask := rocc.module.io.mem.req.bits.mask
```

Comment:
Can you `assert(!(rocc.module.io.mem.req.valid && !rocc.module.io.mem.req.bits.phys))`  Since this path has no TLB, we want to break immediately if the RoCC module generates a virtually addressed memory request.